includeNoKotlin.gradle

```diff
@@ -65,7 +65,7 @@ if (System.getenv("JTRANCS_DEPLOY") != null) {
 			mavenDeployer {
 				beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-				repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+				repository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
```

Comment:
According to [this guide](http://central.sonatype.org/pages/ossrh-guide.html#accessing-repositories), it seems the answer is likely to be, well, no.